TestingTool/actions.py

```diff
@@ -41,10 +41,19 @@ def input_element(action_args, browser_instance):
     data.sendKeys(input_text)
     return True, {"success": "Element from xpath took the received value"}
 
+def go_to_url(action_args, browser_instance):
+    result, driver = element_from_xpath(action_args, browser_instance)
+    go_to_url = action_args.get("go_to_url")
+    if go_to_url is False :
```

Comment:
go_to_url nu va fi false niciodata, verifica ce face functia .get a dictionarelor in python